napari/components/viewer_model.py

```diff
@@ -55,7 +56,6 @@ def __init__(
             title=Event,
             interactive=Event,
             cursor=Event,
-            reset_view=Event,
```

Comment:
I use this to catch reset view when synchronize two instances of viewer. It should be deprecated, not removed.